diff --git a/node_modules/@0x/sol-compiler/lib/src/compiler.js b/node_modules/@0x/sol-compiler/lib/src/compiler.js
index 05e3bfd..971cca9 100644
--- a/node_modules/@0x/sol-compiler/lib/src/compiler.js
+++ b/node_modules/@0x/sol-compiler/lib/src/compiler.js
@@ -485,6 +485,9 @@ var Compiler = /** @class */ (function () {
         if (solcVersion.startsWith('0.6')) {
             return new solc_wrapper_v06_1.SolcWrapperV06(solcVersion, this._opts);
         }
+        if (solcVersion.startsWith('0.7')) {
+            return new solc_wrapper_v06_1.SolcWrapperV06(solcVersion, this._opts);
+        }
         throw new Error("Missing Solc wrapper implementation for version " + solcVersion);
     };
     Compiler.prototype._persistCompiledContractAsync = function (contractPath, currentArtifactIfExists, sourceTreeHashHex, contractName, solcVersion, sourcesByPath, compilerInput, compilerOutput, importRemappings) {
diff --git a/node_modules/@0x/sol-compiler/lib/src/utils/compiler.js b/node_modules/@0x/sol-compiler/lib/src/utils/compiler.js
index 3ef5201..110a748 100644
