diff --git a/node_modules/@0x/sol-compiler/lib/src/compiler.js b/node_modules/@0x/sol-compiler/lib/src/compiler.js
index 1e02adf..cca8254 100644
--- a/node_modules/@0x/sol-compiler/lib/src/compiler.js
+++ b/node_modules/@0x/sol-compiler/lib/src/compiler.js
@@ -328,6 +328,9 @@ class Compiler {
         if (solcVersion.startsWith('0.7')) {
             return new solc_wrapper_v07_1.SolcWrapperV07(solcVersion, this._opts);
         }
+        if (solcVersion.startsWith('0.8')) {
+            return new solc_wrapper_v07_1.SolcWrapperV07(solcVersion, this._opts);
+        }
         throw new Error(`Missing Solc wrapper implementation for version ${solcVersion}`);
     }
     _persistCompiledContractAsync(contractPath, currentArtifactIfExists, sourceTreeHashHex, contractName, solcVersion, sourcesByPath, compilerInput, compilerOutput, importRemappings) {
