diff --git a/node_modules/@openzeppelin/contracts/math/SafeMath.sol b/node_modules/@openzeppelin/contracts/math/SafeMath.sol
index 80490ef..4b2a49d 100644
--- a/node_modules/@openzeppelin/contracts/math/SafeMath.sol
+++ b/node_modules/@openzeppelin/contracts/math/SafeMath.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity ^0.6.0;
+pragma solidity ^0.7.0;
 
 /**
  * @dev Wrappers over Solidity's arithmetic operations with added overflow
diff --git a/node_modules/@openzeppelin/contracts/math/SignedSafeMath.sol b/node_modules/@openzeppelin/contracts/math/SignedSafeMath.sol
index ab87c27..07ddd13 100644
--- a/node_modules/@openzeppelin/contracts/math/SignedSafeMath.sol
+++ b/node_modules/@openzeppelin/contracts/math/SignedSafeMath.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity ^0.6.0;
+pragma solidity ^0.7.0;
 
 /**
  * @title SignedSafeMath
