diff --git a/node_modules/@openzeppelin/contracts/math/SafeMath.sol b/node_modules/@openzeppelin/contracts/math/SafeMath.sol
index 7975c63..a378390 100644
--- a/node_modules/@openzeppelin/contracts/math/SafeMath.sol
+++ b/node_modules/@openzeppelin/contracts/math/SafeMath.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 /**
  * @dev Wrappers over Solidity's arithmetic operations with added overflow
diff --git a/node_modules/@openzeppelin/contracts/math/SignedSafeMath.sol b/node_modules/@openzeppelin/contracts/math/SignedSafeMath.sol
index c8b1bc5..5206fbc 100644
--- a/node_modules/@openzeppelin/contracts/math/SignedSafeMath.sol
+++ b/node_modules/@openzeppelin/contracts/math/SignedSafeMath.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 /**
  * @title SignedSafeMath
